src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -0,0 +1,147 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+
+import {t} from '../../../locale';
+import ApiMixin from '../../../mixins/apiMixin';
+import AsyncView from '../../asyncView';
+import Confirm from '../../../components/confirm';
+import IndicatorStore from '../../../stores/indicatorStore';
+import SettingsPageHeader from '../components/settingsPageHeader';
+
+const TeamRow = createReactClass({
+  displayName: 'TeamRow',
+
+  propTypes: {
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    team: PropTypes.object.isRequired,
+    access: PropTypes.object.isRequired,
+    onRemove: PropTypes.func.isRequired,
+  },
+
+  mixins: [ApiMixin],
+
+  getInitialState() {
+    return {
+      loading: false,
+      error: false,
+    };
+  },
+
+  handleRemove() {
+    if (this.state.loading) return;
+
+    let loadingIndicator = IndicatorStore.add(t('Saving changes..'));
+    let {orgId, projectId, team} = this.props;
+    this.api.request(`/projects/${orgId}/${projectId}/teams/${team.slug}/`, {
+      method: 'DELETE',
+      success: (d, _, jqXHR) => {
+        this.props.onRemove();
+        IndicatorStore.remove(loadingIndicator);
+      },
+      error: () => {
+        this.setState({
+          error: true,
+          loading: false,
+        });
+        IndicatorStore.remove(loadingIndicator);
+      },
+    });
+  },
+
+  render() {
+    let team = this.props.team;
+    return (
+      <tr>
+        <td>
+          <h5 style={{marginBottom: 5}}>{team.name}</h5>
+        </td>
+        {this.props.access.has('project:write') && (
+          <td style={{textAlign: 'right'}}>
+            <Confirm
+              message={t('Are you sure you want to remove this team?')}
+              onConfirm={this.handleRemove}
+              disabled={this.state.loading}
+            >
+              <a className="btn btn-sm btn-default">
+                <span className="icon icon-trash" /> &nbsp;{t('Remove')}
+              </a>
+            </Confirm>
+          </td>
+        )}
+      </tr>
+    );
+  },
+});
+
+class ProjectTeams extends AsyncView {
+  getEndpoints() {
+    let {orgId, projectId} = this.props.params;
+    return [['teams', `/projects/${orgId}/${projectId}/teams/`]];
+  }
+
+  handleRemovedTeam(removedTeam) {
+    this.setState({
+      teams: this.state.teams.filter(team => {
+        return team.slug !== removedTeam.slug;
+      }),
+    });
+  }
+
+  renderEmpty() {
+    return (
+      <div className="box empty-stream">
+        <span className="icon icon-exclamation" />
+        <p>{t('There are no teams with access to this project.')}</p>
+      </div>
+    );
+  }
+
+  renderResults() {
+    let {orgId, projectId} = this.props.params;
+    let access = new Set(this.props.organization.access);
+    return (
+      <div className="panel panel-default horizontal-scroll">
```

Comment:
Can we use <Panel> component like in https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/settings/account/apiApplications.jsx#L157?